specs/core/0_beacon-chain.md

```diff
@@ -1218,7 +1219,7 @@ def process_deposit(state: BeaconState,
     Process a deposit from Ethereum 1.0.
     Note that this function mutates ``state``.
     """
-    proof_of_possession_data = DepositInput(
+    deposit_input = DepositInput(
```

Comment:
I want the name here to signal that it is the data that was signed for the proof of possession (namely, the DepositInput with an empty signature)

Reply:
open to name changes but `depost_input` isn't descriptive enough imo

Reply:
Right, it also took me a couple seconds to understand that `DepositInput.proof_of_possession` is set to `EMPTY_SIGNATURE` instead of the given `proof_of_possession`. What do you think about extracting a  `validate_proof_of_possession` function like: https://github.com/ethereum/py-evm/blob/56792003f676508c58371ebdc58aa0911aede717/eth/beacon/deposit_helpers.py#L43-L70

Reply:
yeah, I like that

Reply:
but still think `deposit_input` isn't aptly named in that context

Reply:
or at least needs an explanatory comment

Reply:
I changed back to `proof_of_possession_data`. 😅